spec/spec_helper.rb

```diff
@@ -11,7 +11,6 @@
 
 RSpec.configure do |config|
   config.before(:suite) do
-    Authy.api_uri = 'http://sandbox-api.authy.com'
     Authy.api_key = 'bf12974d70818a08199d17d5e2bae630'
```

Comment:
I'm guessing this API key should be replaced too but I'll let @senekis chime in there.

Reply:
Good point, this should probably be something that is more obviously an example.